lang/LangPrimSource/PyrArrayPrimitives.cpp

```diff
@@ -2369,12 +2383,142 @@ int prArrayUnlace(struct VMGlobals *g, int numArgsPushed)
 	return errNone;
 }
 
+extern bool compiledOK;
+PyrSymbol* symb_unixCmdAction;
+
+static void string_popen_thread_func(struct sc_process *process)
```

Comment:
if you use a static function form another translation unit, remove the const. the declaration could then either go to a header file, or you can just declare it in the source file where you are going to use it.